sys/fido2/ctap/transport/Kconfig

```diff
@@ -4,4 +4,9 @@
 # General Public License v2.1. See the file LICENSE in the top level
 # directory for more details.
 
+menuconfig MODULE_FIDO2_CTAP_TRANSPORT
+    bool "FIDO2 ctap transport"
```

Comment:
It would be good to unify prompts, in hid transport it is all uppercase. I think `FIDO2 CTAP transport` looks better.